services/catarse/spec/models/billing/payment_spec.rb

```diff
@@ -56,4 +56,34 @@
       it { is_expected.to validate_numericality_of(:installments_count).is_equal_to(1) }
     end
   end
+
+  describe '#before_save' do
```

Comment:
**Sugestão**
Com a mudança para o `validate`, isso vai para dentro do `describe 'Validations'`. E quando vamos testar validations, não podemos fazer só um `expect(obj.save).to be_falsey`, pois não conseguimos garantir que a falha ao salvar foi relacionado com nossa validação. Temos que verificar se o objeto está com o erro que era pra estar devido a falha na validação. Também não é necessário criar um pagamento só pra isso, um `Billing::Payment.new` preenchendo os campos desejados já basta pra testar esse caso.